valhalla/jawn/src/managers/request/RequestManager.ts

```diff
@@ -184,4 +194,30 @@ export class RequestManager extends BaseManager {
       });
     });
   }
+
+  async getRequestAssetById(
+    requestId: string,
+    assetId: string
+  ): Promise<Result<HeliconeRequestAsset, string>> {
+    const { data: assetData, error: assetError } = await getRequestAssetById(
```

Comment:
Can we not name this the same as the function above? Or can we scope it.  it is pretty confusing imo